### ISSUE TITLE
Error using in typescript

### ISSUE THREAD

**Issue by codyaray** [coinbase/gdax-node#301]
Getting this error when importing gdax
```
$ tsc gdax.ts
node_modules/gdax/index.d.ts(225,56): error TS1016: A required parameter cannot follow an optional parameter.
```

Commenting out this line makes this work:
```
cancelAllOrders(args?: { product_id: string }, callback: callback<string[]>): void;
```

Here's the `gdax.ts` file:
```
import Gdax = require('gdax');
const websocket = new Gdax.WebsocketClient(['BTC-USD', 'ETH-USD']);

websocket.on('message', data => {
  console.log(data);
});
websocket.on('error', err => {
  console.log(err);
});
websocket.on('close', () => {
  console.log('closing');
});
```

**Comment by codyaray** [comment]
Thanks, #303 works for me. :)

**Comment by codyaray** [comment]
I'm running tsc 2.8.1, fwiw.

**Comment by fb55** [comment]
Fixed in #303. Thanks, @blair!

**Comment by Kurry** [comment]
@fb55 Can we leave this open until its published?

**Comment by fb55** [comment]
Published this as 0.8.0
